docker/templates/macros.m4

```diff
@@ -327,21 +327,27 @@ lsof
 maven
 openjdk-8-jdk
 openjdk-8-jre
-python2
-python2-dev
 rabbitmq-server
-virtualenv
-')
+# sir dependencies
+libz-dev
+libpq-dev
+libffi-dev
+libssl-dev
+libxml2-dev
+libxslt1-dev
```

Comment:
Copied these from the sir Dockerfile. But I suspect they aren't needed, because they've been there since before the Python v3 upgrade, and weren't ever added here...